src/units/user/dbus-broker.service.in

```diff
@@ -8,7 +8,7 @@ Requires=dbus.socket
 Conflicts=shutdown.target
 
 [Service]
-Type=notify
+Type=notify-reload
 Sockets=dbus.socket
 ExecStart=@bindir@/dbus-broker-launch --scope user
 ExecReload=@bindir@/busctl --user call org.freedesktop.DBus /org/freedesktop/DBus org.freedesktop.DBus ReloadConfig
```

Comment:
ExecReload needs to be dropped, and `dbus-broker/src/units/system/dbus-broker.service.in` needs to be updated too

Reply:
It's up to @dvdhrm to decide as he's the maintainer